letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -500,6 +500,7 @@ Bootstrap() {
     echo "You will need to bootstrap, configure virtualenv, and run pip install manually."
     echo "Please see https://letsencrypt.readthedocs.org/en/latest/contributing.html#prerequisites"
```

Comment:
I wonder if we should also improve that error message?  Right now that link is (1) to readthedocs; (2) is a remnant of the ancient era when using.rst walked the user through venv creation.  Maybe https://certbot.eff.org/docs/contributing.html#running-a-local-copy-of-the-client would be a better link?